commands/jail.js

```diff
@@ -2,39 +2,41 @@ const { SlashCommandBuilder, ButtonBuilder, ButtonStyle, ActionRowBuilder, Embed
 const rulesKimoState = require('../patterns/rulesKimoState');
 const Jail = require('../models/jailTracker');
 const jail = require('../patterns/jail');
+const { jailedRoleID, adminRoleID } = require('../ids.json');
 
 module.exports = {
     data: new SlashCommandBuilder()
-    .setName('jail')
-    .setDescription('jail someone.')
-    .addUserOption(option =>
-        option
-            .setName('target')
-            .setDescription('to jail')
-            .setRequired(true))
-    .addIntegerOption(option =>
-        option
-            .setName('time')
-            .setDescription('minutes to jail')
-            .setRequired(true))
-    .addStringOption(option =>
-        option
-            .setName('reason')
-            .setDescription('reason for jailing')
-            .setRequired(true)),
+        .setName('jail')
+        .setDescription('jail someone.')
+        .addUserOption(option =>
+            option
+                .setName('target')
+                .setDescription('to jail')
+                .setRequired(true))
+        .addIntegerOption(option =>
+            option
+                .setName('time')
+                .setDescription('minutes to jail')
+                .setRequired(true))
+        .addStringOption(option =>
+            option
+                .setName('reason')
+                .setDescription('reason for jailing')
+                .setRequired(true)),
 
     async execute(interaction, client) {
 
-        if (!interaction.member.roles.cache.get('1203377553763475497')) return interaction.reply({ content: `you cannot use this...`, ephemeral: true });
+        if (!interaction.member.roles.cache.get(adminRoleID)) 
```

Comment:
I find single line ifs can be a little tricky to read if you aren't on an ultrawide, so moving the return to the following line sometimes looks a lil better